docs-public/example-runner.js

```diff
@@ -106,6 +109,9 @@ var ExampleRunner = (function() {
             });
 
             system.config({
+                paths: {
+                    'raven-js': npmUrl + "/raven-js"
+                },
```

Comment:
Is this necessary? All other modules seem to be functional without declaring paths.